### PR TITLE
Feature/3552 list item tag example clean up

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # BLUI-3552

#### Changes proposed in this Pull Request:
- ListItemTag
Examples
Should have an example for the different variants
Should have an example for colors

#### Screenshots:
Screenshot 2022-10-12 at 3 52 45 PM

#### To Test:
- yarn start:reactdev

### CODE REVIEW

File: docs/src/componentDocs/ListItemTag/markdown/ListItemTagExamples.mdx [hunk 1]
```diff
@@ -1,12 +1,19 @@
-import { BasicListItemTag } from '../examples/BasicListItemTag';
-import { ComplexListItemTag } from '../examples/ComplexListItemTag';
+import { ListItemTag, ListItemTagUsingInfoListItem, ListItemTagWithVariants } from '../examples';
 
-# Basic List Item Tag
+## List Item Tag
 
-<BasicListItemTag />
+A `<ListItemTag>` is a stylized Typography element used for labelling. The text and background colors are customizable via the `fontColor` and `backgroundColor` props.
```
Comment (emclaug2) [inline docs/src/componentDocs/ListItemTag/markdown/ListItemTagExamples.mdx:5]: Typo "Labeling"

**Comment by emclaug2** [comment]
@bkarambe if you merge dev into this branch, it'll fix the failed ci step.

**Comment by emclaug2** [comment]
See this: https://github.com/brightlayer-ui/react-component-library/pull/601

After that gets merged in & then you merge dev -> feature, it'll start working 👍

**Review by emclaug2 — APPROVED** [review]

**Comment by daileytj** [comment]
Screen Shot 2022-10-12 at 1 09 25 PM

Are these 2 items supposed to be represented as tags? e.g., `<InfoListItem>`

**Review by daileytj — APPROVED** [review]
See my one comment. I'm not sure on the syntax there, but if that's correct then this is good to go. Great work!

**Pull request merged**